test/e2e/cluster_extension_install_test.go

```diff
@@ -322,6 +257,81 @@ func TestClusterExtensionInstallSuccessorVersion(t *testing.T) {
 	}, pollDuration, pollInterval)
 }
 
+func TestClusterExtensionInstallReResolvesWhenNewCatalog(t *testing.T) {
+	t.Log("When a cluster extension is installed from a catalog")
+	t.Log("It resolves again when a new catalog is available")
+
+	// Tag the image with the new tag
+	baseImage := "localhost:30000/e2e/test-catalog:v1"
+	var err error
+	err = crane.Tag(baseImage, "latest", crane.Insecure)
```

Comment:
Should `"latest"` be parsed from the value of `testLatestCatalogRefEnvVar` (instead of being hardcoded)?

Reply:
`testLatestCatalogRefEnvVar` consist of the complete image path with latest tag. We just need to tag name and not the complete image name here

Reply:
Right, but if we change that environment variable value to use a different tag, we'd have to change all the hardcoded `"latest"` references in the e2e code as well. If we parse the tag out of the reference, there's just the one source of truth (but also see my other comment about possibly refactoring the envvars).